CefSharp.BrowserSubprocess/Program.cs

```diff
@@ -41,8 +40,29 @@ public static int Main(string[] args)
             }
 
             Debug.WriteLine("BrowserSubprocess shutting down.");
-
+            
             return result;
         }
+
+        private static void AwaitParentProcessExit(int parentProcessId) 
+        {
+            try 
+            {
+                var parentProcess = Process.GetProcessById(parentProcessId);
+                parentProcess.WaitForExit();
+            }
+            catch (Exception e) 
+            {
+                //main process probably died already, bailout
+                Debug.WriteLine(e);
+                return;
+            }
+
+            Task.Delay(1000); //wait a bit before exiting
+
+            Debug.WriteLine("BrowserSubprocess shutting down forcibly.");
```

Comment:
Should probably log to the native `CEF Logging`, trivial to add an wrapper method.

Reply:
I just copied the same pattern that was already there.